### DISCUSSION TITLE
(Question) Adding bounds to items

### DISCUSSION THREAD

**Discussion by Ahmed-Ateeq** [samuelarbibe/dnd-timeline#44] [Q&A] [answered]
How do i put a resize limit to any of my items. For example, I don't want the item to exceed 1 hour. How can I achieve that?

**Comment by samuelarbibe** [comment]
Hey!  
If you want to limit the resize, 
you can put a logic inside the `onResizeEnd` callback that checks if the updated relevance exceeds the allowed duration, and if so skip the actual state update.

Keep in mind, this will still allow the client to drag and resize the item larger than the limit, but the update will not actually register and change state, so it will just pop back to the unchanged position.

**Reply by Ahmed-Ateeq** [reply]
hello, ah yeah I was doing that but I was hoping that somehow I could limit the drag and resize movements for the users so they don't move it out of bounds at the first place.

**Reply by samuelarbibe** [reply, marked as answer]
Unfortunately, limiting drag and resize animations is not currently supported.  
The reason is that resizing and dragging is handled by css translations for performance purposes, and intervening with that could cause redundant renders at the cost of performance.
